webpack.config.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
```

Comment:
We have 3 `webpack` config files. `webpack.common.js`, `webpack.dev.js`, `webpack.prod.js`. It seems you are missing those files in your work. Please kindly confirm.